### ISSUE TITLE
low precision and and recall

### ISSUE THREAD

**Issue by natxopedreira** [dog-qiuqiu/Yolo-FastestV2#49]
Hello

Im training with only one class from coco dataset, data file is standar only changes anchors and classes to 1

```
[name]
model_name=coco

[train-configure]
epochs=300
steps=150,250
batch_size=128
subdivisions=1
learning_rate=0.001

[model-configure]
pre_weights=model/backbone/backbone.pth
classes=1
width=352
height=352
anchor_num=3
anchors=8.54,20.34, 25.67,59.99, 52.42,138.38, 103.52,235.28, 197.43,103.53, 238.02,287.40

[data-configure]
train=coco_person/train.txt
val=coco_person/val.txt
names=data/coco.names
```

I get an AP of 0.41 but with low precision 0.53 and recall of 0.41 that makes that model prediction has lots of false positives.

Why im getting that low precision and recall?

PD. i checked bbox annotations and are correct

Thanks!

**Comment by natxopedreira** [comment]
@dog-qiuqiu can you please comment what is a good expected value for precision and recall in this implementation?

I tried increasing the dataset, argumenting, increase image size for training... and running evaluate the best i can get is 
`Precision:0.754652 Recall:0.563240 AP:0.660795 F1:0.645046`

**Comment by InputBlackBoxOutput** [comment]
I am working with a custom dataset with one class. Decreasing the learning rate to 0.0005 and training for 500 epochs gave a better evaluation result

**Comment by natxopedreira** [comment]
What values are you getting¿ 

I also used this one with great results, not as fast but more accurate
[https://github.com/ppogg/YOLOv5-Lite](https://github.com/ppogg/YOLOv5-Lite)
